src/components/course/data/constants.js

```diff
@@ -99,17 +105,21 @@ export const DISABLED_ENROLL_REASON_TYPES = {
   SUBSCRIPTION_SEATS_EXHAUSTED_NO_ADMINS: 'subscription_seats_exhausted_no_admin',
   SUBSCRIPTION_LICENSE_NOT_ASSIGNED: 'subscription_license_not_assigned',
   SUBSCRIPTION_LICENSE_NOT_ASSIGNED_NO_ADMINS: 'subscription_license_not_assigned_no_admin',
+  OFFER_EXPIRED: 'offer_expired',
 };
 
 /* eslint-disable max-len */
 export const DISABLED_ENROLL_USER_MESSAGES = {
+  [DISABLED_ENROLL_REASON_TYPES.SUBSIDY_NOT_ACTIVE]: REASON_USER_MESSAGES.SUBSIDY_NOT_ACTIVE || REASON_USER_MESSAGES.SUBSIDY_NOT_ACTIVE_NO_ADMINS,
+  [DISABLED_ENROLL_REASON_TYPES.POLICY_NOT_ACTIVE]: REASON_USER_MESSAGES.POLICY_NOT_ACTIVE || REASON_USER_MESSAGES.POLICY_NOT_ACTIVE_NO_ADMINS,
```

Comment:
Since `can-redeem` also determines if there exist an admin user associated with the enterprise and tailors the message accordingly, the `||` is to capture cases where the API would return the `NO_ADMIN` vs admin use case as to avoid the proptypes error message. Since the returned reason is identical regardless, they are tied to the same key hence the `||`

Reply:
After our discussion around the prop-type, ended up removing these entirely since we are dependent on the API returning these specific error messages.